server/routes.js

```diff
@@ -46,21 +46,6 @@ module.exports = [
   },
   {
     method: 'GET',
-    path: '/resources',
```

Comment:
Please could you leave this route? We are using it for getting the reviewed resources.